app-cryptography/signing-party/autobuild/defines

```diff
@@ -0,0 +1,11 @@
+PKGNAME=signing-party
+PKGSEC=misc
+PKGDES="Various OpenPGP related tools made for key signing party"
+PKGDEP="gnupg perl-class-methodmaker perl-mailtools perl-mime-tools perl-net-idn-encode perl-term-readkey perl-text-template \
+	perl-gnupg-interface imagemagick python-3 perl-gd qrencode qprint"
+BUILDDEP="perl gnupg"
+# TeXLive: for LaTeX support in gpg-key2latex.
```

Comment:
If an error occurs without texlive installed, you should be using PKGRECOM.

Reply:
Will do.